doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -871,8 +871,8 @@ PyArrayIter_Type
    .. c:member:: npy_intp *PyArrayIterObject.backstrides
 
        How many bytes needed to jump from the end of a dimension back
-       to its beginning. Note that *backstrides* [k]= *strides* [k]*d
-       *ims_m1* [k], but it is stored here as an optimization.
+       to its beginning. Note that `backstrides`[k]= `strides`[k] *
+       `dims_m1`[k], but it is stored here as an optimization.
```

Comment:
Note also for future reference that bare single backticks are basically never right in vanilla Sphinx. They assign some text the "default role". If a particular build has been configured to set this to something useful, and you know what that is and it's local convention to use it, then great, but otherwise it's better to be explicit :-). (This is definitely the most annoying incompatibility between markdown and rst.)